libraries/DataFlash/DataFlash_MAVLinkLogTransfer.cpp

```diff
@@ -155,13 +173,17 @@ void DataFlash_Class::handle_log_request_end(GCS_MAVLINK &link, mavlink_message_
     mavlink_msg_log_request_end_decode(msg, &packet);
```

Comment:
Yes, I explicitly thought about this and left it so any GCS could cancel the download.